File: Resources/Python/sort.py
def sort_array(arr):
    """
    This function takes an array as input and returns a sorted version of it.
    """
    if not arr:
        print("The array is empty.")
        return arr

    sorted_arr = sorted(arr)
    return sorted_arr

if __name__ == "__main__":
    array = [34, 7, 23, 32, 5, 62]
    
    print("Original Array:", array)
    
    sorted_array = sort_array(array)
    
    print("Sorted Array:", sorted_array)

File: Resources/C/sort.c
#include <stdio.h>

void sortArray(int arr[], int n)
{
    for (int i = 0; i < n - 1; i++)
    {
        for (int j = 0; j < n - i - 1; j++)
        {
            if (arr[j] > arr[j + 1])
            {
                int temp = arr[j];
                arr[j] = arr[j + 1];
                arr[j + 1] = temp;
            }
        }
    }
}

void printArray(int arr[], int n)
{
    for (int i = 0; i < n; i++)
    {
        printf("%d ", arr[i]);
    }
    printf("\n");
}

int main()
{
    int array[] = {34, 7, 23, 32, 5, 62};
    int n = sizeof(array) / sizeof(array[0]);

    printf("Original Array: ");
    printArray(array, n);

    sortArray(array, n);

    printf("Sorted Array: ");
    printArray(array, n);

    return 0;
}
